File: flutter/lib/models/lagu.dart
class Lagu {
  final int id;
  final String judul;
  final String penyanyi;
  final String album;
  final String cover;
  final String file;

  Lagu({
    required this.id,
    required this.judul,
    required this.penyanyi,
    required this.album,
    required this.cover,
    required this.file,
  });

  factory Lagu.fromJson(Map<String, dynamic> json) {
    return Lagu(
      id: json['id'],
      judul: json['judul'],
      penyanyi: json['penyanyi'],
      album: json['album'],
      cover: json['cover'],
      file: json['file'],
    );
  }

  Map<String, dynamic> toJson() {
    return {
      'id': id,
      'judul': judul,
      'penyanyi': penyanyi,
      'album': album,
      'cover': cover,
      'file': file,
    };
  }
}

File: laravel/app/Models/Lagu.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Factories\HasFactory;
use Illuminate\Database\Eloquent\Model;

class Lagu extends Model
{
    use HasFactory;
    protected $table = 'lagu';

    protected $fillable = [
        'judul', 'penyanyi', 'album', 'cover', 'file',
    ];
}